comparing_template/comparing_template/urls.py

```diff
@@ -15,8 +15,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from register import views as v
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
+    path('admin/', admin.site.urls),
+    path("register/", v.register, name="register"),  # <-- added
```

Comment:
Co ma na celu komentarz added? Komentarze powinny wyjaśniać to co robi ta linijka/fragment kodu, jeśli to ma nam powiedzieć, że to dodaliśmy to tego nie chcemy, bo taki komentarz moglibyśmy równie dobrze dać przy wszystkich linijkach w projekcie które dodamy